gnucash/gnome/dialog-sx-since-last-run.c

```diff
@@ -435,6 +435,10 @@ gsslrtma_populate_tree_store (GncSxSlrTreeModelAdapter *model)
             }
         }
 
+        // if there are no instances for the instance skip adding
+        if (g_list_length (instances->instance_list) == 0)
```

Comment:
An empty glist is indicated by GList* being NULL, so `if (instances->instance_list)` does the same with less cost when there are lots of instances.

Also shouldn't this go before populating `last_occur_date_buf`?

Reply:
I think I had a print statement for `last_occur_date_buf` and forgot to move this test after I had removed it.
Changed.